libia2/include/ia2.h

```diff
@@ -149,19 +126,9 @@ asm(".macro mov_pkru_eax pkey\n"
 #define PTRS_PER_PAGE (PAGE_SIZE / sizeof(void *))
 #define IA2_MAX_THREADS (PTRS_PER_PAGE)
 
-#ifdef LIBIA2_INSECURE
-#define pkey_mprotect insecure_pkey_mprotect
-static int insecure_pkey_mprotect(void *ptr, size_t len, int prot, int pkey) {
-  return 0;
-}
-#define INIT_RUNTIME(n)                                                        \
-  int ia2_n_pkeys_to_alloc = 0;                                                \
-  INIT_RUNTIME_COMMON(n)
-#else
 #define INIT_RUNTIME(n)                                                        \
   int ia2_n_pkeys_to_alloc = n;                                                \
   INIT_RUNTIME_COMMON(n)
```

Comment:
nit: `INIT_RUNTIME_COMMON` can be inlined here since it was just the runtime init that was common to both INSECURE and the default.